cli/src/main/kotlin/xyz/block/artifactswap/cli/di/CommonModule.kt

```diff
@@ -53,7 +53,10 @@ private fun createArtifactSwapConfig(properties: Properties): ArtifactSwapConfig
       ),
     protosSchemaVersionProperty =
       getProperty("artifactswap.protosSchemaVersionProperty", defaults.protosSchemaVersionProperty),
-    artifactoryBaseUrl = getProperty("artifactswap.artifactoryBaseUrl", defaults.artifactoryBaseUrl),
+    artifactoryBaseUrl =
+      getProperty("artifactswap.artifactoryBaseUrl", defaults.artifactoryBaseUrl),
+    bomPublisherBranchName =
+      getProperty("artifactswap.bomPublisherBranchName", defaults.bomPublisherBranchName),
```

Comment:
I think this name is misleading because the bom publisher never reads it, only the downloader does.

maybe `bomSourceBranchName`?